stories/helperComponents/storyComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { TPAComponentsProvider } from '../../src/components/TPAComponentsConfig';
+
+// tslint:disable:react-hooks-nesting
+export const storyComponent = Component => {
```

Comment:
Just a suggestion for the future - we can also pass `options` prop (besides `Component`) to control the mobile enable/disable. Maybe could be useful

Reply:
interesting 🤔 
Maybe it should be part of the `props` received by the component in the line below (line 6)?
So it will look like this:
```jsx
const { mobile, ...rest } = props;
return (
      <div ref={rootRef}>
        <TPAComponentsProvider value={{ mobile, rtl }}>
          <Component {...rest} />
        </TPAComponentsProvider>
      </div>
    );
```
wdyt @nitayneeman ?

Reply:
Sounds good as well 👍